backend/app/tests/api/routes/test_creds.py

```
@@ -407,6 +406,7 @@ def test_multiple_provider_credentials(
     # Create Langfuse credentials
     langfuse_credential = {
         "organization_id": org.id,
+        "project_id": project.id,
```

Ruff (0.12.2)

Undefined name `project`

(F821)